docs/catalog/cloud/use-serverless.md

```diff
@@ -0,0 +1,43 @@
+---
+version: 1.0
+submitted_by: markus-ntt-seidl
+published_date: TBD
+category: cloud
+tags: 
+ - cloud
+ - role:cloud-engineer
+ - size:small
+---
+
+# Use serverless cloud services
+
+## Description
+
+Serverless cloud services are services that the cloud provider manages for the application. These scale dynamically with the workload needed to fulfill the service task and apply best practices to keep resource usage minimal.
+
+## Solution
+
+Adapt serverless services for the application.
```

Comment:
This feels very similar to the PR https://github.com/Green-Software-Foundation/green-software-patterns/pull/144. Should we merge any additional info from here into that PR or do you feel that this is significantly different enough that it should be its own pattern?

Reply:
Ah, I think we need some kind of process to prevent that the same Issue is converted twice.
There was no indication to me that this already existed.

This is the same pattern, as you indicated. So maybe we can merge it.

Reply:
The initial review and SME reviews are supposed to catch duplicates so while its kind of annoying from a PR submitted perspective, it does get caught before being merged. #144 was closed in favor of this entry so I think we can resolve this specific conversation